home.php

```diff
@@ -15,31 +15,22 @@
 	<div class="wrap">
 		<?php
 		// Only show the feature and subfeatures when not paged.
-		if ( ! is_paged() ) {
-			if ( have_posts() ) {
-				$post_count = 0;
-
-				while ( have_posts() ) {
-					the_post();
-
-					$post_count++;
+		if ( ! is_paged() && have_posts() ) {
+			?>
+			<div class="wrap__item wrap__item--full-width">
+				<?php get_template_part( 'templates/entry/featured' ); ?>
+				<hr>
```

Comment:
Since the featured template is only used once, can the `<hr>` be added to the template? Or rather, could the effect be achieved with CSS instead?